brave/browser/password_manager/brave_password_manager_client.cc

```diff
@@ -339,6 +339,20 @@ BravePasswordManagerClient::GetPasswordStore() const {
              profile_, ServiceAccessType::EXPLICIT_ACCESS).get();
 }
 
+#if defined(SAFE_BROWSING_DB_LOCAL)
```

Comment:
It was removed in https://github.com/brave/muon/commit/8316ae196eee53a7774945458ea86446ced19556#diff-11a9200d30980e9b495a6efb898420af